sc/DC_expMatrix_deeper.characterization.pickle.info

```diff
@@ -0,0 +1,25 @@
+{
+    "collection": "GEO",
+    "description": "A data set of 1244 cells from human blood samples. The included cell types are The included cell types are dendritic cells (DCs) with mutants over expressed for marker genes CD141+, CD1C+, pathogenic cells driving blastic plasmacytoid dendritic cell neoplasm (BPDCN) from four donors, a double negative mutant CD11C+/CD141-/CD1C-, monocytes and plasmacytoid DCs, and cells FACS sorted for AXL6+/SIGLEC+ forming a new DC subplopulation.  GEO Accession: GSE94820.",
```

Comment:
The included cell types are The included cell types

Also, the second sentence is very hard to read (would be better with e.g. enumeration).
We should discuss formatting options - I think using html in descriptions should already work, but I am not sure we want to have too much html code in these (so far simple) text fields. @astaric ?